CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # bedrock-ledger-consensus-continuity ChangeLog
 
+## 8.0.1 - TBD
+
+### Removed
+- `axios` was removed from `lib/client.js` in order to move away from axios.
+- `axios` was removed from `package.json`
+
+### Added
+- `@digitalbazaar/http-client` was added to the project.
+
+### Changed
```

Comment:
I would think that just
```suggestion
### Changed
- Replaced `axios` with `@digitalbazaar/http-client`
```
would suffice, but unsure if others think the details are more useful.

Reply:
@mandyvenables I'll let @mattcollier decide on this. He had issues with my CHANGELOG notes in a previous PR.